src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -117,13 +118,17 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
  * @param expectedValues  array of dictionaries containing the expected values in the same format as
  *                       attribute read completion handler. Requires MTRAttributePathKey values.
  *                       See MTRDeviceResponseHandler definition for dictionary details.
+ *                       The expectedValues and expectedValueInterval arguments need to be both
+ *                       nil or both specified within the expected range, or will be both ignored.
+ *
  * TODO: document better the expectedValues is how this command is expected to change attributes when read, and that the next
  * readAttribute will get these values
  *
  * @param expectedValueInterval  maximum interval in milliseconds during which reads of the attribute will return the value being
- * written. This value will be clamped to timeout
+ * written. This value must be within [1, UINT32_MAX], and will be clamped to this range.
```

Comment:
So what will _actually_ happen here is that values <= 0 cause the expectedValue bits to be ignored, while values > UINT32_MAX are clamped.  So neither this comment nor the one above about "or both specified within the expected range, or will be both ignored" is right.  How about:
```suggestion
 * written. If the value is less than 1, both this value and expectedValues will be ignored.
            If this value is greater than UINT32_MAX, it will be clamped to UINT32_MAX.
```